sound/soc/sof/intel/skl.c

```diff
@@ -88,6 +88,7 @@ const struct snd_sof_dsp_ops sof_skl_ops = {
 	/* DAI drivers */
 	.drv		= skl_dai,
 	.num_drv	= SOF_SKL_NUM_DAIS,
+
```

Comment:
is this blank line intentional?

Reply:
and should there be a rom timeout in had-loader-skl as well? @zhuyingjiang can you chime in?

Reply:
@plbossart this timeout doesnt seem to be used in SKL. @zhuyingjiang could you please confirm if it is needed or not? If not, maybe I can remove it altogether

Reply:
@plbossart @ranj063 Yes, SKL still need this timeout. As I test, the HDA_ROM_INIT_TIMEOUT which is 70ms is some short, So I use the already defined HDA_DSP_INIT_TIMEOUT 500ms, If this can be platform specific, 150 ms can work for SKL.

Reply:
@zhuyingjiang in that case, will you post a patch to use this 150ms value set for rom_init_value?

Reply:
@ranj063 OK

Reply:
@ranj063 #717 is it.